test/node/int_32_tests.js

```diff
@@ -6,7 +6,9 @@ const expect = require('chai').expect;
 
 describe('Int32', function() {
   describe('Constructor', function() {
-    var value = 42;
+    const hexValue = '0x2A';
```

Comment:
Should be ` const hexValue = 0x2A;` (no quotes)

Reply:
@emadum We're using string because the test is checking to see if the `+` properly converts the string hex value to base 10, which it does. Try for yourself in REPL:

```js
> +"0x2a"
42
```